scripts/covid_hosp_prod.R

```diff
@@ -2,39 +2,44 @@
 #
 # Ran into some issues with targets:
 #   https://github.com/ropensci/targets/discussions/666#discussioncomment-9050772
+#
 
 source("scripts/targets-common.R")
 
-
-#' Get exclusions from a JSON file for a given date
-#'
-#' @param date A date
-#' @param exclusions_json A JSON file with exclusions in the format:
-#'
-#'    {"exclusions": {"2024-03-24": "ak,hi"}}
-get_exclusions <- function(date, exclusions_json = here::here("scripts", "geo_exclusions.json")) {
-  s <- jsonlite::read_json(exclusions_json)$exclusions[[as.character(date)]]
-  if (!is.null(s)) {
-    return(s)
+insufficient_data_geos <- c("as", "gu", "mp", "vi")
+forecast_generation_date <- as.character(seq.Date(as.Date("2024-04-22"), Sys.Date(), by = "1 week"))
+# forecast_generation_date <- as.character(seq.Date(as.Date("2024-01-01"), Sys.Date(), by = "1 week"))
```

Comment:
I guess setting it to 4-22 gives a more realistic picture of when it's starting to run?